src/CMakeLists.txt

```diff
@@ -211,9 +211,13 @@ if(AV_EIGEN_MEMORY_ALIGNMENT)
     AddCompilerFlag("-faligned-new")
   elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 6.0)
     AddCompilerFlag("-faligned-new")
+  elseif(CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10.0)
+    AddCompilerFlag("-faligned-new")
   elseif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 19.12)
     AddCompilerFlag("/Zc:alignedNew")
   else()
+      message(FATAL_ERROR "ZZZ ${CMAKE_CXX_COMPILER_ID} ${CMAKE_CXX_COMPILER_VERSION}")
```

Comment:
?

Reply:
This is debugging code, sorry about this.

Reply:
Fixed now.